### DISCUSSION TITLE
presidio sometimes misses very crucial information such telephone number

### DISCUSSION THREAD

**Discussion by sadathknorket** [microsoft/presidio#1474] [Q&A]
Given that I used this tutorial https://github.com/microsoft/presidio/blob/main/docs/samples/python/example_pdf_annotation.ipynb

and this pdf example 
[personalpdf.pdf](https://github.com/user-attachments/files/17493676/personalpdf.pdf)

The outputted pdf was 
[output_annotated.pdf](https://github.com/user-attachments/files/17493680/output_annotated.pdf)

We could notice telephone number was missed to be annotated .

Is there any extra parameter or something I could do to avoid such mistakes ?

**Comment by omri374** [comment]
Hi, for telephone numbers specifically, Presidio is configured to search for numbers belonging to some countries but not all. See this issue, where the `supported_regions` param is updated: https://github.com/microsoft/presidio/issues/1140

For addresses and others, I would suggest to look into transformers based models. See this example (still in PR) for a full presidio customization: https://github.com/microsoft/presidio-research/blob/omri/evaluator_updates/notebooks/5_Evaluate_Custom_Presidio_Analyzer.ipynb

**Comment by sadathknorket** [comment]
How does it know about the region ? Through the phone number itself ?

**Reply by omri374** [reply]
The user has to provide the regions it wants it to scan. There is no region detection here. If your data likely has Turkish phone numbers, you should add Turkey to the list of supported regions.
